packages/runtime/src/enhancements/create-enhancement.ts

```diff
@@ -120,13 +123,26 @@ export function createEnhancement<DbClient extends object>(
 
     let result = prisma;
 
-    if (hasPassword === undefined || hasOmit === undefined) {
+    if (
+        process.env.ZENSTACK_TEST === '1' || // avoid caching in tests
```

Comment:
It turns out the test failure is due to the cache of these flags. I've added the condition to exclude the caching in test env.

Reply:
Thank you!